manifests/2.4.1/opensearch-2.4.1.yml

```diff
@@ -101,3 +101,12 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: neural-search
+    repository: https://github.com/opensearch-project/neural-search.git
+    ref: tags/2.4.0.0
```

Comment:
Need change this ref.

Reply:
Done!

Reply:
Thanks! Also be sure to change the newly added repo refs.